lib/src/app.dart

```diff
@@ -143,12 +143,7 @@ class App {
   /// Removes the user's local credentials. This will also close any associated Sessions.
   /// 
   /// If [user] is null logs out [currentUser] if it exists.
-  Future<void> logout(User? user) async {
-    user ??= currentUser;
-    if (user == null) {
-      return;
-    }
-
+  Future<void> logout([User? user]) async {
```

Comment:
I think `user` being optional here is a mistake, but I guess this point is moot if we decide to remove the method altogether.

Reply:
let's remove in a follow-up PR if we decide to